deploy.yml

```diff
@@ -1771,4 +1796,5 @@ parameters:
 - name: IMAGE_TAG
   value: latest
 - name: IMAGE
-  value: quay.io/redhat-services-prod/hcc-devprod-tenant/ephemeral-namespace-operator
+  value: quay.io/cloudservices/ephemeral-namespace-operator
```

Comment:
This image needs to be reverted. The original is the one that gets used by Konflux.